purchases/src/main/kotlin/com/revenuecat/purchases/utils/EntitlementInfoHelper.kt

```diff
@@ -0,0 +1,23 @@
+package com.revenuecat.purchases.utils
+
+import com.revenuecat.purchases.Store
+import java.util.Date
+
+internal class EntitlementInfoHelper private constructor() {
```

Comment:
I think you can make this an object
```suggestion
internal object EntitlementInfoHelper {
```
Then, you will need to remove the `companion object` below.

Reply:
oh right